tests/integration/openshift-ci-create-pod.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
What is this file about?

Reply:
I will remove that one. I saved it for debugging openshift ci .